magento/magento2-page-builder
# Removes less code that uses frontend definitions in an adminhtml file…

…. Fixes #775

### Description (*)
This removes references to less classes in an adminhtml less file that are only defined in the frontend of Magento's Blank and/or Luma themes.
So you aren't able to use these references in an adminhtml file and really shouldn't as it won't work.
See https://github.com/magento/magento2-page-builder/issues/775#issuecomment-913747559 for an example (I did investigate all the other problems by now and they are all the same problem).

When comparing the resulting css file before and after this change, zero changes were found, so the PR shouldn't cause any issues.

### Story
?

### Bug
?

### Task
?

### Fixed Issues (if relevant)
1. #775

### Builds
?

### Related Pull Requests
?

### Manual testing scenarios (*)
1. Setup a Magento installation (version 2.4.3 for example) and have the page builder module installed and enabled
2. Install the [official less.js compiler](https://lesscss.org/): `npm install less@3.13.1`
3. Run `rm -R var/view_preprocessed/* pub/static/*; bin/magento setup:static-content:deploy -f --theme=Magento/backend en_US`
4. Validate the generated less code by linting the outputted files. Especially the `styles.less` file is important in scope of this PR: `node_modules/.bin/lessc -l var/view_preprocessed/pub/static/adminhtml/Magento/backend/en_US/css/styles.less`, notice that you get a bunch of warnings.
5. Make a backup of the `pub/static/adminhtml` directory
6. Apply the changes from this PR
7. Run step 3 and 4 again, notice that the warnings are gone now.
8. Compare the contents of the `pub/static/adminhtml` directory with the backup you created in step 5. There will be no differences.

End result: cleaner code, no warnings anymore, reduced code to maintain, reduced confusion when people try to understand what your code is doing.

### Questions or comments
It would be nice if all Magento less code in all kinds of various repo's could get tested and validated by the [official less.js compiler](https://lesscss.org/) in some kind of static test so problematic code like this doesn't get added to the codebase.

(Very much unrelated, but [your template for contributions](https://raw.githubusercontent.com/magento/magento2-page-builder/develop/.github/PULL_REQUEST_TEMPLATE.md) isn't really friendly to the open source community since it asks for a lot of references to your private Jira board, which we can't provide ...)

### Checklist
